lib/utils/object_utils.js

```diff
@@ -43,3 +44,16 @@ function values(object) {
   }
   return values;
 }
+
+function keys(object) {
```

Comment:
The point of `::values` is to provide what ES2016 will. So that when it's available, this function will be removed. We should just remove `::keys` and use `Object::keys` here.